V/VMEC/build_tarballs.jl

```diff
@@ -65,11 +65,10 @@ products = [
 # Dependencies that must be installed before this package can be built
 dependencies = [
     # MbedTLS is an indirect dependency, fix the version for building 
-    BuildDependency(PackageSpec(name = "MbedTLS_jll", version = v"2.24.0")),
-    Dependency("MPICH_jll"; platforms=filter(!Sys.iswindows, platforms)),
-    Dependency("OpenBLAS_jll"),
+    BuildDependency(PackageSpec(name = "MbedTLS_jll")),
```

Comment:
libblastrampoline works well only with Julia v1.7+, and there is also the problem that libblastrampoline 3 (the one used in Julia v1.7) and 5 (the one you're using to build here) have incompatible ABIs: https://github.com/JuliaRegistries/General/pull/64542#issuecomment-1189195017

Reply:
Fixed to work with 1.7, I saw an example in [P/Pfapack](https://github.com/JuliaPackaging/Yggdrasil/P/Pfapack/build_tarballs.jl) of how to do this for 1.7 and 1.8, should I be following that?

Reply:
I removed the libblastrampoline dependence for now.